src/bin/nydus-image/main.rs

```diff
@@ -264,6 +264,14 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
                         .help("Set the size of data chunks, must be power of two and between 0x1000-0x1000000:")
                         .required(false),
                 )
+                .arg(
+                    Arg::new("batch-size")
+                        .long("batch-size")
+                        .help("Set the size of batch data chunks, must be power of two, between 0x1000-0x1000000 or be zero:")
```

Comment:
Set the batch size to merge small chunks, must be power of two, between 0x1000-0x1000000 or be zero.

Reply:
Changed.